src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,38 @@
+import {NgModule} from "@angular/core";
+import {CarSearchComponent} from "./car-search/car-search.component";
+import {ReactiveFormsModule} from "@angular/forms";
+import {AsyncPipe, NgClass, NgForOf, NgIf} from "@angular/common";
+import {WidgetComponent} from "./widget/widget.component";
+import {WeatherWidgetContentComponent} from "./widget/weather-widget/weather-widget.component";
+import {SprintWidgetContentComponent} from "./widget/sprint-widget/sprint-widget.component";
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatIconModule} from '@angular/material/icon';
+import {MatButtonModule} from '@angular/material/button';
+import {MatListModule} from '@angular/material/list';
+
+@NgModule({
+  imports: [
+    ReactiveFormsModule,
+    NgIf,
+    MatToolbarModule,
+    MatListModule,
+    MatButtonModule,
+    MatIconModule,
+    NgForOf,
+    AsyncPipe,
+    NgClass
+  ],
+  declarations: [
+    CarSearchComponent,
```

Comment:
we use shared module only to use shared modules, we don't declare components here